src/UserAccountDataChecker.php

```diff
@@ -24,13 +24,13 @@ class UserAccountDataChecker extends DataChecker {
 	public $mUser = null;
 
 	public function __construct() {
-		$this->addCheck( array( &$this, 'checkDomainValidity' ), array() );
-		$this->addCheck( array( &$this, 'checkDomainUser' ), array() );
-		$this->addCheck( array( &$this, 'checkCreatePermissions' ), array() );
-		$this->addCheck( array( &$this, 'checkSorbs' ), array() );
-		$this->addCheck( array( &$this, 'checkUserExists' ), array() );
-		$this->addCheck( array( &$this, 'checkPasswordLength' ), array() );
-		$this->addCheck( array( &$this, 'checkEmailValidity' ), array() );
+		$this->addCheck( array( $this, 'checkDomainValidity' ), array() );
```

Comment:
The `$that` variable could be any other literal but since we use `$this` (which in PHP has a specific meaning by holding an instance reference for the current object) then we can also use `$that` as a simple variable identifier.